src/main/java/com/gregtechceu/gtceu/api/item/tool/GTToolType.java

```diff
@@ -351,13 +351,14 @@ public class GTToolType {
             .electric(GTValues.IV)
             .toolClasses(GTToolType.WRENCH)
             .build();
-    public static final GTToolType BUZZSAW = GTToolType.builder("buzzsaw")
+    public static final GTToolType BUZZSAW_LV = GTToolType.builder("lv_buzzsaw")
+            .idFormat("lv_%s_buzzsaw")
```

Comment:
Renaming the items will make everyone's buzzsaws disappear. Don't.

Reply:
How likely is it that anyone crafted a "lv" buzzsaw with an IV tool though?

Reply:
Doesn't matter

Reply:
> How likely is it that anyone crafted a "lv" buzzsaw with an IV tool though? Edit: I'll happily change it. Just my opinion tho

It also doesn't just affect items crafted, it would ripple into any kubejs scripts /datapacks / Addon Mods as well.
Breaking changes like that would probably be more suited for the Neoforge Branch

Reply:
> > How likely is it that anyone crafted a "lv" buzzsaw with an IV tool though? Edit: I'll happily change it. Just my opinion tho
> 
> It also doesn't just affect items crafted, it would ripple into any kubejs scripts /datapacks / Addon Mods as well. Breaking changes like that would probably be more suited for the Neoforge Branch

Noted. Will keep in mind :)

Reply:
@Ghostipedia Though, Won't this break already crafted IV tier SD and BZ's either way?